src/pages/config/ConfigDetailsChangesPage.tsx

```diff
@@ -24,15 +25,18 @@ export function ConfigDetailsChangesPage() {
   }
 
   return (
-    <ConfigLayout
-      basePath={`configs/${id}`}
-      isConfigDetails
-      title={<ConfigsDetailsBreadcrumbNav configId={id} />}
-      isLoading={isLoading}
-    >
-      <div className="flex flex-col items-start">
-        <ConfigChangeHistory data={historyData ?? []} isLoading={isLoading} />
-      </div>
-    </ConfigLayout>
+    <>
+      <Head prefix="Config Details Changes" />
```

Comment:
It would be nice if we used config name here when available, something `Config Name Changes`

Reply:
Noted